src/xz/private.h

```diff
@@ -75,3 +89,7 @@
 #ifdef HAVE_DECODERS
 #	include "list.h"
 #endif
+
+#if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
This is needed only if VS2013 support is really wanted. `_snprintf` doesn't conform to C99 while `snprintf` in VS2015 should conform. So there is a minor risk to use `_snprintf`.

Reply:
Fixed in a new commit.
Added a function `msvc_snprintf()` to emulate C99 `snprintf()` with `_snprintf()`.
This can be removed when VS2013 support is dropped.